src/components/input-field/examples/input-field-textarea.tsx

```diff
@@ -57,13 +67,15 @@ export class InputFieldTextareaExample {
         this.value = event.detail;
     }
 
-    private setEnabled(event: CustomEvent<boolean>) {
-        event.stopPropagation();
-        this.disabled = event.detail;
+    private toggleEnabled() {
+        this.disabled = !this.disabled;
+    }
+
+    private toggleReadonly() {
+        this.readonly = !this.readonly;
     }
 
-    private setRequired(event: CustomEvent<boolean>) {
-        event.stopPropagation();
-        this.required = event.detail;
+    private toggleRequired() {
+        this.required = !this.required;
```

Comment:
I guess this was changed to make it all look the same when you copied `toggleReadonly` from the `input-field-text` example? I think the other pattern is better than the one this was changed to, so I'm going to change it back. I'll change the `input-field-text` example too though.

Reply:
yes I stole it from the other example 🙈

Reply:
I've updated the examples a bunch now.